### DISCUSSION TITLE
NonPlanar slice capability request

### DISCUSSION THREAD

**Discussion by truegret** [OrcaSlicer/OrcaSlicer#2729] [Ideas]
As a 3D print enthusiast, hobbyist, and engineer I would like to see the ability to slice certain models/objects with NonPlanar features. This would enhance the esthetics of some models. With the availability of custom fan shrouds and extended hot-end nozzles, I believe this would be a desirable capability and feature.

I am particularly interested in the application of NonPlanar slicing and printing for air foils.

I have implemented such capability a few years ago, with a Linux environment build of Slic3r. However, that was a rather cumbersome endeavor that makes the risk/challenge/struggle greater than the reward. I recently discover this fork and have embraced it. Kudos to the whole team. I have used Slic3r, Repetier, Simplify3D, Prusa, Cura, and IdeaMaker as each bring a little something different.

I look forward to a discussion around the practicality of incorporating NonPlanar slicing into the SoftFever flavor of the OrcaSlicer, and ultimate implementation.

**Comment by oliverbrossmann** [comment]
I would love this. It would add strength for certain parts that would significantly improve the product's life.
